app/Task.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
```

Comment:
Takodje, treba da se postavi veza izmedju user-a i task-a.
Task pripada samo jednom user-u. :) Pogledaj po dokumentaciji kako to da postignes.